modules/openapi-generator/src/main/resources/typescript-node/model.mustache

```diff
@@ -2,7 +2,7 @@
 {{#models}}
 {{#model}}
 {{#tsImports}}
-import { {{classname}} } from './{{filename}}';
+import { {{classname}} } from '{{filename}}';
```

Comment:
I removed the `./` from the template and added it to the `toFileName()` method in the generator. Having the prefix here makes it impossible to have a correct import for an external dependency, like:

```typescript
import { MyThing } from '@mynamespace/mypackage/mything';
```